pages/about.html

```diff
@@ -4,9 +4,20 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- <link href="styles/style.css" rel="stylesheet"> -->
+  <link href="styles/about.css" rel="stylesheet">
+  
   <title>Document</title>
```

Comment:
You can update this title tag to be specific to you page. It could be like "About Hoku" -- this is what will show up in the tab of your browser